core/src/main/java/de/danoeh/antennapod/core/util/ShareUtils.java

```diff
@@ -50,11 +50,15 @@ private static String getItemShareText(FeedItem item) {
 		return item.getFeed().getTitle() + ": " + item.getTitle();
 	}
 
+    public static boolean hasLinkToShare(FeedItem item) {
```

Comment:
I think this should also be moved to `FeedItemUtil`

Reply:
`ShareUtils#hasLinkToShare()` is basically a wrapper over `FeedItemUtil` . 

Semantically, the current codes centralizes the behavior that for sharing feed item link, it will fallback to using feed's link in `ShareUtils`. For example, if it is decided sharing feed item link's behavior needs to be changed again, the change will be isolated within `ShareUtils` , rather than spreading it around.